src/localization.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable */
+(function (document) {
+  const page = window.location.toString().includes('login')
+    ? 'login'
+    : 'register';
+
+  const supportedLanguages = ['en', 'fi'];
```

Comment:
Wow, lot of work.. Imo we made a slight error when we decided to implement this in vanilla.
But now its there.. =)

Reply:
Yeah perhaps.. 😄 But it has been fun to do for a change!